aws-rds-customdbengineversion/src/main/java/software/amazon/rds/customdbengineversion/Translator.java

```diff
@@ -0,0 +1,120 @@
+                .status(model.getStatus())
+                .build();
+    }
+
+
+    public static DeleteCustomDbEngineVersionRequest deleteCustomDbEngineVersion(final ResourceModel model) {
+        return DeleteCustomDbEngineVersionRequest.builder()
+                .engine(model.getEngine())
+                .engineVersion(model.getEngineVersion())
+                .build();
+    }
+
+    public static List<software.amazon.awssdk.services.rds.model.Tag> translateTagsToSdk(final Collection<Tag> tags) {
+        return Optional.ofNullable(tags).orElse(Collections.emptyList())
+                .stream()
+                .map(tag -> software.amazon.awssdk.services.rds.model.Tag.builder()
+                        .key(tag.getKey())
+                        .value(tag.getValue())
+                        .build()
+                )
+                .collect(Collectors.toList());
+    }
+
+    public static Map<String, String> translateTagsToRequest(final Collection<Tag> tags) {
+        return Optional.ofNullable(tags).orElse(Collections.emptyList())
+                .stream()
+                .collect(Collectors.toMap(Tag::getKey, Tag::getValue));
+    }
+
+    private static <T> Stream<T> streamOfOrEmpty(final Collection<T> collection) {
+        return Optional.ofNullable(collection)
+                .map(Collection::stream)
+                .orElseGet(Stream::empty);
+    }
+
+    public static List<Tag> translateTagsFromSdk(final Collection<software.amazon.awssdk.services.rds.model.Tag> sdkTags) {
+        return streamOfOrEmpty(sdkTags)
+                .map(tag -> Tag
+                        .builder()
+                        .key(tag.key())
+                        .value(tag.value())
+                        .build()
+                )
+                .collect(Collectors.toList());
+    }
+
+    static ResourceModel translateToModel(
+            final DBEngineVersion engineVersion
+    ) {
+        return ResourceModel.builder()
+                .databaseInstallationFilesS3BucketName(engineVersion.databaseInstallationFilesS3BucketName())
+                .databaseInstallationFilesS3Prefix(engineVersion.databaseInstallationFilesS3Prefix())
+                .description(engineVersion.dbEngineVersionDescription())
+                .engine(engineVersion.engine())
+                .engineVersion(engineVersion.engineVersion())
+                .kMSKeyId(engineVersion.kmsKeyId())
+                .tags(translateTagsFromSdk(engineVersion.tagList()))
+                .dBEngineVersionArn(engineVersion.dbEngineVersionArn())
+                .build();
+    }
+
+    public static DescribeDbEngineVersionsRequest describeDbEngineVersionsRequest(final String nextToken) {
+        return DescribeDbEngineVersionsRequest.builder()
+                .marker(nextToken)
+                .build();
+    }
+
+    public static List<ResourceModel> translateToModel(final Stream<DBEngineVersion> engineVersionsStream) {
```

Comment:
I think we've been sticking to `translateToSdk` and `translateFromSdk` kind of naming pattern. You might consider applying it here for the sake of consistency.